src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -207,6 +214,38 @@ export default {
 			return this.parent && this.$store.getters.message(this.token, this.parent)
 		},
 
+		conversation() {
+			return this.$store.getters.conversations[this.token]
+		},
+
+		participant() {
+			const participantIndex = this.$store.getters.getParticipantIndex(this.token, this.$store.getters.getParticipantIdentifier())
+			if (participantIndex !== -1) {
+				return this.$store.getters.getParticipant(this.token, participantIndex)
+			}
+
+			return {
+				inCall: PARTICIPANT.CALL_FLAG.DISCONNECTED,
+			}
+		},
+
+		messagesList() {
+			return this.$store.getters.messagesList(this.token)
+		},
+
+		isLastCallStartedMessage() {
+			const messages = this.messagesList
+			const lastCallStartedMessage = messages.reverse().find((message) => message.systemMessage === 'call_started')
+			return lastCallStartedMessage ? (this.id === lastCallStartedMessage.id) : false
+		},
```

Comment:
Actually I'm not sure if this should be done inside the message...
Could also imagine doing it in the MessageList where `messagesList()` is also already used.